Applications/DataExplorer/Base/Color.cpp

```diff
@@ -23,18 +23,17 @@
 
 namespace GeoLib {
 
-Color* getRandomColor()
+Color getRandomColor()
 {
 	std::array<unsigned char, 3> col;
```

Comment:
minor comment: Isn't it possible to write here `Color col;`? just to improve readability bit... At the beginning I thought Color has a constructor accepting std::array.